<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmXvFuncAsm</class>
 <widget class="QWidget" name="FrmXvFuncAsm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>50</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>50</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>70</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scAreaXvFuncAsm">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scAreaWcXvFuncAsm">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>48</width>
        <height>520</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="XMatHLine" name="hLine1"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="XMatToolButton" name="btnXvFuncUp">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatToolButton" name="btnXvFuncDown">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="XMatHLine" name="hLine2"/>
     </item>
     <item>
      <widget class="XMatToolButton" name="btnShowXvFuncAsm">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>XMatToolButton</class>
   <extends>QToolButton</extends>
   <header>XMatToolButton.h</header>
  </customwidget>
  <customwidget>
   <class>XMatHLine</class>
   <extends>QWidget</extends>
   <header>XMatHLine.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
